src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/Pages/Error.cshtml

```diff
@@ -16,8 +16,11 @@
 
 <h3>Development Mode</h3>
 <p>
-    Swapping to <strong>Development</strong> environment will display more detailed information about the error that occurred.
+    Swapping to the <strong>Development</strong> environment displays detailed information about the error that occurred.
 </p>
 <p>
-    <strong>Development environment should not be enabled in deployed applications</strong>, as it can result in sensitive information from exceptions being displayed to end users. For local debugging, development environment can be enabled by setting the <strong>ASPNETCORE_ENVIRONMENT</strong> environment variable to <strong>Development</strong>, and restarting the application.
+    <strong>The Development environment shouldn't be enabled for deployed applications</strong>.
+    It can result in displaying sensitive information from exceptions to end users.
+    For local debugging, enable the <strong>Development</strong> environment by setting the <strong>ASPNETCORE_ENVIRONMENT</strong> environment variable to <strong>Development</strong>
+    and restarting the app or calling <code>UseEnvironment("Development")</code> on the host in <strong>Program.Main</strong> and restarting the app.
```

Comment:
Is this actually good advice? I think we must not recommend this pattern because people will leave it checked in to their code. I strongly urge that we NOT add this recommendation.

Reply:
And I'll bring in the @blowdart hammer if I must!

Reply:
Good point. Using the env var should be our guidance.

Reply:
When you put it that way I agree, no need for @blowdart.

Reply:
Too late. I will be bringing back a wombat to beat you with next week.